microsite/data/plugins/jenkins-actions.yaml

```diff
@@ -0,0 +1,12 @@
+---
+title: Scaffolder jenkins client actions
+author: robertonav20
+authorUrl: https://github.com/robertonav20
+category: Scaffolder
+description: Collection of actions to integrate backstage template with jenkins. For example job creation, run, deletion and so on.
+documentation: https://github.com/backstage/community-plugins/tree/main/workspaces/jenkins/plugins/scaffolder-backend-module-jenkins
+iconUrl: https://img.icons8.com/color/1600/jenkins.png
```

Comment:
Is there a better place for us to link out to an image here? Or can we just copy the jenkins logo into this repo? :pray:

Reply:
I don't know how the site is working... In case we decide to proceed with store locally the logo how could I reference, just with relative path? There is an example?

Reply:
Here's an example: https://github.com/backstage/backstage/blob/master/microsite/data/plugins/score-card.yaml#L8

Reply:
Hi @benjdlambert,
now should be ok